### ISSUE TITLE
Google-Search Problem with Azure, and Problem by showing google answer

### ISSUE THREAD

**Issue by fadiaismael** [mckaywrigley/chatbot-ui#766]
Google Search does not work with Azure
And when this is fixed, google answers are not displayed correctly, instead the user gets client-side error

**Comment by devnjw** [comment]
Does google search works with openai api? Mine doesn't..

**Comment by grski** [comment]
#767 fixed it for me, running Azure OpenAI gpt-35-turbo
![image](https://github.com/mckaywrigley/chatbot-ui/assets/16924479/20f61862-ce86-4e1b-933d-bc9fd9d56907)

**Comment by swiftdev29** [comment]
> #767 fixed it for me, running Azure OpenAI gpt-35-turbo
> ![image](https://user-images.githubusercontent.com/16924479/241657321-20f61862-ce86-4e1b-933d-bc9fd9d56907.png)

Still wont work for me. Using OpenAI GPT 4

**Comment by studydev** [comment]
From the /pages/api/google.ts file to Google search results, I confirmed that it works normally.
However, when querying OpenAI for search results, the branch statement for Azure is not processed and the code is written to operate only for OpenAI.
This problem can be solved by changing the code to branch OpenAI and Azure as if the URL and header information were in /pages/api/models.ts.
I was able to confirm that it works.
<img width="1057" alt="image" src="https://github.com/mckaywrigley/chatbot-ui/assets/5790864/6900691d-04e8-464c-8126-8dadcde8d4eb">
